packages/ui/src/components/molecules/BlockInteractionMessage/BlockInteractionMessage.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {StyleSheet, Dimensions} from 'react-native';
+import {useConfig} from '../../../config/ConfigContext';
+import {BlockInteractionScreen, Card, Icon} from '../../atoms';
+import WarningCard from '../WarningCard/WarningCard';
+
+/**
+ * @description To use this component, please use setBlockInteractionConfig({visible:boolean,message:'string',callback:()=> {},iconName(optional):'string'}) from useConfig of aos-mobile/ui
+ */
+
+const BlockInteractionMessage = ({}) => {
+  const {blockInteractionConfig, setBlockInteractionConfig} = useConfig();
+  if (!blockInteractionConfig?.visible) {
+    return null;
+  }
+
+  const handleButton = () => {
+    blockInteractionConfig?.callback();
+    setBlockInteractionConfig({
+      visible: false,
+      message: '',
+      callback: () => {},
+    });
+  };
+
+  return (
+    <BlockInteractionScreen hideHeader={true}>
+      <Card style={styles.container}>
+        <WarningCard
+          errorMessage={blockInteractionConfig.message}
+          style={styles.warningCard}
+        />
+        <Icon
+          name={blockInteractionConfig?.iconName}
+          touchable={true}
+          onPress={handleButton}
+        />
```

Comment:
I think we can do better than just one icon in the middle 😄 

As I already told you, I think we should enable people to give a list of actions as, depending on the context, we can have multiple actions possible on this pop-up : refresh the app, disconnect, try again, ... 

So my suggestion is to have three props in blockInteractionConfig :
- visible : to know if component should be visible
- message : to fill in WarningCard
- actions : a list of actions to display composed with : 
```
{
    iconName?: string
    tittle: string
    onPress: () => void
    color?: Color
}
```
and this list should be displayed with Button, or IconButton depending on presence of iconName, in a column style, button should take most of the width of the pop-up like it's done in `PopupApplicationNotConfigured`